### PR TITLE
Adds Colab part and minor changes to ml_ops/stage2/get_started_bqml_training notebook

### PR DESCRIPTION
**Pull request by krishr2d2**
Opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under `# Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

Opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under the `# Community Notebooks` section, pointing to the author or the author's team.
- [x] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).

### CODE REVIEW

File: notebooks/community/ml_ops/stage2/get_started_bqml_training.ipynb [hunk 1]
```diff
@@ -38,6 +38,11 @@
         "      View on GitHub\n",
```
Comment (andrewferlitsch) [inline notebooks/community/ml_ops/stage2/get_started_bqml_training.ipynb:38]: <p>You should be able to move deleting the temporary BQ table out of the delete_storage.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/491/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Comment (krishr2d2) [reply notebooks/community/ml_ops/stage2/get_started_bqml_training.ipynb:38]: Done !!

**Comment by krishr2d2** [comment]
While conducting test runs on Colab and Workbench, the newly added model registry part gave permission errors on Colab while trying to update iam-policy bindings. This case anyhow was already described above the cell suggesting to grant some additional permissions to set the corresponding policy bindings.
stage2_bqml_newly_added_code

**Comment by andrewferlitsch** [comment]
Automated tests not run. Submitter manually tested in Colab and Workbench. I am manually merging the PR.

**Pull request merged**